keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -206,8 +208,19 @@ endpoints.
 
 #### Future Work
 
-After OpenAPI v3 is implemented, we can start working on updating clients to
-consume OpenAPI v3. A separate KEP will be published for that.
+After OpenAPI v3 is implemented, we can start working on updating
+clients to consume OpenAPI v3. A separate KEP will be published for
+them. One example is [OpenAPI V3 for kubectl
+explain](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/3515-kubectl-explain-openapiv3/README.md).
```

Comment:
I'd mention the work that @seans3  is doing here. As in, it's not really so much into the future as much as what we're doing right now.